### DISCUSSION TITLE
AA Registered Group ID

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#585] [Ideas]
Would it be possible to add a GSO Registered Group ID field? Ideally, it would pop up when toggling on the field "part of a group" when either adding/editing a meeting. It should not be a required field. 

I do not want to install another plugin such as the one described in #584 

Thanks

**Comment by tech2serve** [comment]
For whatever reason, I can’t find #584. 

In the past when this has been brought up, the consensus of the developers is the keep TSML focused on the meetings and the list, and not try to solve more general intergroup/district/CSO management tasks. The easiest solution is to put the number and other data in the Group notes field.

This isn’t to say we won’t revisit this as our group grows and we have more bandwidth.

**Comment by dougie-wougie** [comment]
Thanks. I have been adding them to the Group notes since I do not know where to add them.
